libminifi/src/FlowController.cpp

```diff
@@ -163,6 +174,31 @@ FlowController::~FlowController() {
   provenance_repo_ = nullptr;
 }
 
+bool FlowController::applyConfiguration(std::string &configurePayload) {
```

Comment:
You can use civet to run a webserver. I have a commit that can help if that is useful for you.